### PR TITLE
Show level briefing before the tutorial instead of after

### LINKED ISSUE #2001: Level Briefing should show before tutorial
**What feature or enhancement would you like to add?**
It doesn't make much sense to show the level 1 brief after the tutorial in the campaign. Instead, it should be at the start before the tutorial begins.

**How would this feature benefit the gameplay of CorsixTH?**
Makes the level ordering better and more consistent.

**Was the feature in the original game?**
Can't remember

**What version of CorsixTH are you using?**
 0.65

**Does this feature/enhancement relate to a current problem?**
No

**Additional information**
Add any other information or screenshots about the feature request here.

### PR DESCRIPTION
**Pull request by tobylane**
*Fixes #2001*

**Describe what the proposed change does**
- Always show the level briefing on day 1
- Don't show the final message of the tutorial when playing level 1, as it is the level briefing already shown

### CODE REVIEW

File: CorsixTH/Lua/game_ui.lua [hunk 1]
```diff
@@ -1099,8 +1099,10 @@ tutorial_phases = {
           {_S.introduction_texts["level15"]},
           {_S.introduction_texts["level16"]},
           {_S.introduction_texts["level17"]},
-          {_S.introduction_texts["level1"]},
         }
+        if TheApp.world.map.level_number ~= 1 then
+          table.insert(texts, _S.introduction_texts["level1"])
+        end
```
Comment (lewri) [inline CorsixTH/Lua/game_ui.lua:1105]: Should we also cover the demo's tutorial? Or does the demo one make more logical sense to come after?

Comment (tobylane) [reply CorsixTH/Lua/game_ui.lua:1104]: Yes, both of them before is better.

**Comment by lewri** [comment]
I haven't been able to test this currently -- but can you confirm if moving the level 1/demo briefing to before the tutorial is triggering the initial opening countdown timer? Do we want the tutorial to be timed?

**Comment by tobylane** [comment]
> can you confirm if moving the level 1/demo briefing to before the tutorial is triggering the initial opening countdown timer? 

It does, that needs fixing.

**Review by lewri — APPROVED** [review]
Tested. I'm happy.

**Pull request merged**